service/src/main/kotlin/fi/espoo/evaka/invoicing/service/InvoiceGenerator.kt

```diff
@@ -126,7 +120,10 @@ class InvoiceGenerator(
                 .toSet() +
                 permanentPlacements.keys +
                 temporaryPlacements.values.flatMap { pairs -> pairs.map { it.second.child.id } }
-        val operationalDaysByChild = tx.getOperationalDatesForChildren(range, allChildren)
+        val operationalDaysByChild =
+            tx.getOperationalDatesForChildren(range, allChildren).mapValues {
+                DateSet.of(it.value.map(LocalDate::toFiniteDateRange))
```

Comment:
might be more readable to add an overload for DateSet.of which accepts LocalDates directly